### PR TITLE
Correct broken curves asset links

### PR DESCRIPTION
**Pull request by JosepBove**
Repairing a broken link

## Description

I changed the broken link to the valid one

## Related Issues

#52316 

## Tests

I added the following tests:

Not test needed

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.
- [x] The analyzer (`flutter analyze --flutter-repo`) does not report any problems on my PR.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Did any tests fail when you ran them? Please read [Handling breaking changes].

- [x] No, no existing tests failed, so this is *not* a breaking change.
- [ ] Yes, this is a breaking change. *If not, delete the remainder of this section.*
   - [ ] I wrote a design doc: https://flutter.dev/go/template *Replace this with a link to your design doc's short link*
   - [ ] I got input from the developer relations team, specifically from: *Replace with the names of who gave advice*
   - [ ] I wrote a migration guide: *Replace with a link to your migration guide*

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Test Coverage]: https://github.com/flutter/flutter/wiki/Test-coverage-for-package%3Aflutter
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Handling breaking changes]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes

### CODE REVIEW

File: packages/flutter/lib/src/animation/curves.dart [hunk 1]
```diff
@@ -1334,7 +1334,7 @@ class Curves {
   /// By default is the curve used to animate pages on iOS back to their original
   /// position if a swipe gesture is ended midway through a swipe.
   ///
-  /// {@animation 464 192 https://flutter.github.io/assets-for-api-docs/assets/animation/fast_linear_to_slow_ease_in.mp4}
+  /// {@animation 464 192 https://flutter.github.io/assets-for-api-docs/assets/animation/curve_fast_linear_to_slow_ease_out.mp4}
```
Comment (goderbauer) [inline packages/flutter/lib/src/animation/curves.dart]: It looks like this file is actually incorrectly named. While it is showing the right curve, it should actually be named "curve_fact_linear_to_slow_in.mp4". The curve is generated here: https://github.com/flutter/assets-for-api-docs/blame/master/packages/diagrams/lib/src/curve.dart#L475

Could you correct the naming there and then use the correct file name here?

Comment (JosepBove) [reply packages/flutter/lib/src/animation/curves.dart]: Yeah, I will commit it

Comment (JosepBove) [reply packages/flutter/lib/src/animation/curves.dart]: I have commited the correct name here and also created a pull request in the curve generator. (https://github.com/flutter/assets-for-api-docs/pull/111)

**Comment by JosepBove** [comment]
I have commited the correct name here and also created a pull request in the curve generator. (flutter/assets-for-api-docs#111)

**Comment by JosepBove** [comment]
I have commited the correct name and a fix for `linear_to_ease_out` here and also created a pull request in the curve generator to fix the incorrect naming at the generator. flutter/assets-for-api-docs#111

This should be ready now

**Comment by goderbauer** [comment]
Looks good, but before we can submit this, we will have to get flutter/assets-for-api-docs#111 ready and submitted.

**Comment by goderbauer** [comment]
@gspencergoog or @jcollins-g This got me thinking, would it actually be possible to hook into the dartdoc generation process somewhere to check that link targets actually exist?

**Comment by JosepBove** [comment]
You can submit this PR, flutter/assets-for-api-docs#111 has been merged.

**Review by goderbauer — APPROVED** [review]
LGTM

Thanks.

**Pull request merged**